aws/ssm/ecs.go

```diff
@@ -23,7 +25,9 @@ func StartEcsSession(ctx context.Context, config aws.Config, region, cluster, ta
 	}
 
 	target := ssm.StartSessionInput{
-		Target: aws.String(fmt.Sprintf("ecs:%s_%s_%s", cluster, taskId, containerName)),
+		DocumentName: docName,
```

Comment:
What is the DocumentName? (just for my knowledge)

Reply:
The DocumentName refers to a SSM Document - a definition for an automation in the AWS Systems Manager service.
Usually, you set up the SSM Agent on an EC2 box and you can administer the box through these automation.

AWS provides a large catalog of documents along with the ability to bring your own automations.

The most relevant "DocumentNames" for what we're doing is found in [aws/ssm/get_document_name.go](https://github.com/nullstone-io/nullstone/pull/74/files#diff-3bec2a0484b3e6adb41f7ac4bbc6c56f2724041a83f3b78a3ef81f2c1a621ea4)
```
var (
	DocNameStartSSH                = "AWS-StartSSHSession"
	DocNamePortForward             = "AWS-StartPortForwardingSession"
	DocNamePortForwardToRemoteHost = "AWS-StartPortForwardingSessionToRemoteHost"
)
```

Reply:
Ah, seems like a similar naming convention to Policy "Documents". Got it, thanks!